assets/src/application/ui/forms/modalForm/RenderModalForm.tsx

```diff
@@ -1,8 +1,16 @@
-import React from 'react';
+import React, { useCallback, useMemo } from 'react';
 import { __ } from '@wordpress/i18n';
-import { ButtonProps } from 'antd/lib/button';
 import { SaveOutlined } from '@ant-design/icons';
-import { Modal } from 'antd';
+import {
+	Modal,
+	ModalBody,
+	ModalCloseButton,
+	ModalContent,
+	ModalFooter,
+	ModalHeader,
+	ModalOverlay,
+} from '@infraUI/layout/modal';
```

Comment:
hmmmm... 

having second thoughts about this....

plz see [this comment](https://github.com/eventespresso/event-espresso-core/pull/2682#discussion_r406254235)

I think it may be better to create and utilize an adapter that hides all of this composition but allows components like `RenderModalForm` to configure all of the necessary bits and pieces.

The concern I have is that if we ever need to switch UI libraries again in the future, what are the chances that the new library is going to have ALL of these same (or similar) components:

 - Modal
 - ModalBody
 - ModalCloseButton
 - ModalContent
 - ModalFooter
 - ModalHeader
 - ModalOverlay

For example, what if a replacement UI library does NOT have separate components for `ModalHeader`, `ModalContent`, and `ModalFooter` ???
If we had to do something like manually concatenate all of those things together, then every single implementation throughout our application would have to be updated accordingly. Even if that is only two components its still twice the work than if you only had to perform that update to the adapter.

Sorry for changing directions on you after having already approved your previous PR (although it was YOUR comment that got me thinking about this 😆 )